### DISCUSSION TITLE
Cmáx

### DISCUSSION THREAD

**Discussion by AnaSoco** [Open-Systems-Pharmacology/Forum#1376] [Questions & Problems]
Hi everyone, 

I'm trying to develop a renal impairment model for PIP. I made a simulation (n=1000) considering the renal functions as follows: 

1) Healthy (>90 mL/min/1.73)
2) Moderate RI (30-60 mL/min/1.73)
3) Severe RI (<30 mL/min/1.73)

The elimination phase looks fine; however, I'm not reaching the Cmáx; I already try to increase the resolution settings to > 60 pts/h

Example: Below
I'm new working with PK-Sim; thus, maybe I'm missing something. 

Any advice? 

Thank you! 
Ana 

![Captura de pantalla 2022-11-08 162745](https://user-images.githubusercontent.com/74992003/200606425-f4c78efa-8ab4-46ef-9022-51f2765a0f76.png)

**Comment by StephanSchaller** [comment]
Hi Ana, do you have PK in other populations to check if the initial absorption kinetics are generally well predicted by the model?
Best,
Stephan

**Reply by AnaSoco** [reply]
Hi Stephan, thank you for your reply.

Actually, yes, I made simulations with healthy volunteers. 

Examples below:
 
1) Publication 1 Batra simulation 
![Captura de pantalla Batra ](https://user-images.githubusercontent.com/74992003/200777324-109f9973-ad39-449e-917f-f24351b26a33.png)

2) Publication 2 Chandokar 
![Captura de pantalla Chandokar ](https://user-images.githubusercontent.com/74992003/200777610-ab1bfe64-0bea-442e-b0ac-e61942cd469f.png)


Best regards,
Ana

**Reply by StephanSchaller** [reply]
These are all IV infusions?

How do populations and route of administration differ from the first figure above?

**Reply by AnaSoco** [reply]
Yes, all of them are IV infusions.

In the first figure (renal patient RI)) study from Aronoff 1983, the administration protocol was: 60 mg/Kg of PIP over 2-4 min.
As for the population: 
White-American 
Age: 39-66  
Weight: 60-85 Kg
Height: 167-183
and the eGFR was in a range of 36-48 mL/min/1.73 m2

In the second study from Batra in healthy volunteers (HV), the administration protocol was: Daily PIP 6 g every 6 h over 30 min infusion. Blood samples were collected on day 1.

Population: 
White-American 
Age: 18-52
Weight: 63-97 Kg
Height: 165-185
and the eGFR was > 90 mL/min/1.73 m2

Finally, in the third study from Chandokar (HV), the administration protocol was: Three 4 g doses admon every 6 h over 30 min infusion.

Population: 
European 
Age: 18-50
BMI: ≥18.5 and ≤30 kg/m2
eGFR was > 90 mL/min/1.73 m2


Additionally, to develop the renal impairment model, I created individuals according to the disease state option by PK-Sim V.11; therefore, the following changes are made to de model:

Kidney volume --> HV = 0.44 ; RI = 0.34
Kidney blood flow --> HV = 1.32; RI 0.48
Plasma protein binding --> HV= 0.7; RI = 0.75

Thank you in advance.

Kind regards
Ana

**Reply by StephanSchaller** [reply]
okay, I do not see a good reason why the initial distribution volume should be affected, so I am tending towards declaring the Aronoff data as outlier data (or a population with low blood volume...any additional characteristics to this population (mentioned in the publication) on top of renal impairment?).

**Reply by AnaSoco** [reply]
I just reread the paper, but I can't find any other relevant characteristics in this population; they only mentioned renal impairment and discussed PIP excretion. 

Maybe I should try with another data instead and see how it works.

Thank you, Stephan! 
Best regards 
Ana

**Reply by prvmalik** [reply]
It happens and is an opportunity to advance CKD modeling. See our other paper on OAT substrates in CKD by first author Sam Dubinsky.  Could be related to transporters or a change in body composition associated with CKD.
